src/Vester.sol

```diff
@@ -124,8 +125,8 @@ contract Vester is Initializable, IVester {
         }
         vestingPosition.claimed = true;
         // Claim AURA rewards
-        // TODO: Q: should send all AURA rewards even if there are multiple vesting positions?
         AURA_REWARD_POOL.getReward();
+        emit ClaimedAuraRewards(AURA.balanceOf(address(this)));
```

Comment:
GitHub won't let me make an actual suggestion here because it covers too many unchanged lines, but I'd recommend something like this instead (L129-135):

```solidity
        uint256 auraClaimed = AURA.balanceOf(address(this));
        // Transfer staked AURA BAL to beneficiary
        STAKED_AURABAL.safeTransfer(beneficiary, vestingPosition.amount);
        // Transfer AURA to beneficiary
        AURA.safeTransfer(beneficiary, auraClaimed);

        emit Claimed(_nonce, vestingPosition.amount);
        emit ClaimedAuraRewards(auraClaimed);
```